<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KGraphViewerPreferencesReopenPreviouslyOpenedFilesWidget</class>
 <widget class="QWidget" name="KGraphViewerPreferencesReopenPreviouslyOpenedFilesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>115</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>15</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <layout class="QHBoxLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item>
    <widget class="QGroupBox" name="reopenPreviouslyOpenedFilesMode">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Reopen previously opened files</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="selectedId" stdset="0">
      <number>-1</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QRadioButton" name="yes">
        <property name="text">
         <string>&amp;Yes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="no">
        <property name="text">
         <string>No</string>
        </property>
        <property name="shortcut">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ask">
        <property name="text">
         <string>As&amp;k</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
